src/charts/bar.js

```diff
@@ -207,13 +206,15 @@ define(function(require) {
                     .rangeRound([chartHeight, 0])
                     .padding(0.1);
             }
-            colorScale = d3Scale.scaleOrdinal().range(colorSchema).domain(data.map((_, i) => i))
 
-            let range = colorScale.range();
-            topicColorMap = colorScale.domain().reduce((memo, item, i) => {
-                memo[item] = range[i];
-                return memo;
-            }, {});
+            colorList = data.map(d => d)
```

Comment:
ahh we need it reversed before we use the index in the function below, and we dont want to reverse the data so we need a map first